src/test/java/org/jenkinsci/plugins/workflow/cps/SnippetizerTest.java

```diff
@@ -106,6 +117,24 @@
         assertRoundTrip(new CoreStep(new ArtifactArchiver("x.jar")), "step([$class: 'ArtifactArchiver', artifacts: 'x.jar'])");
     }
 
+    @Test public void coreStepWithSymbol() throws Exception {
+        JUnitResultArchiver aa = new JUnitResultArchiver("target/surefire/*.xml");
+        aa.setAllowEmptyResults(true);
+        assertRoundTrip(new CoreStep(aa), "junit allowEmptyResults: true, testResults: 'target/surefire/*.xml'");
+    }
+
+    @Test public void coreStepWithSymbolWithSoleArg() throws Exception {
+        JUnitResultArchiver aa = new JUnitResultArchiver("target/surefire/*.xml");
+        assertRoundTrip(new CoreStep(aa), "junit 'target/surefire/*.xml'");
```

Comment:
Is this not duplicating stuff already tested (or which should be tested) in `CoreStepTest`?